packages/cedar-amcharts/src/render/render.ts

```diff
@@ -102,6 +113,11 @@ export function fillInSpec(spec: any, definition: any) {
         /* tslint:enable */
         // TODO: map other fields besides value like color, size, etc
 
+        // handle le color
+        if (definition.styles && definition.styles.colors) {
```

Comment:
I wasn't expecting this to be needed so I'm curious what happens if you don't have this? is this for bar graph outlines? or needed for line charts?

Reply:
`graph.lineColor` actually hits all the colors that are needed, both the bar fill and outline. It's also applicable to chart types like line, bubble, etc. The other option that we discussed...I'm trying to remember it....only was applicable to the fill color, not outline on bar charts for example.